kubetest/util.go

```diff
@@ -65,7 +65,11 @@ func insertPath(path string) error {
 // Essentially curl url | writer
 func httpRead(url string, writer io.Writer) error {
 	log.Printf("curl %s", url)
-	r, err := http.Get(url)
+	// TODO: we should probably create only one Transport and reuse it for all calls
```

Comment:
creating it here is kinda gross, but it wasn't clear where a better place might be (maybe `main.go`?). I was more interested in fixing the broken PR tests than figuring out the best location to initialize this.